firecracker.go

```diff
@@ -26,7 +26,16 @@ import (
 	ops "github.com/firecracker-microvm/firecracker-go-sdk/client/operations"
 )
 
-const firecrackerRequestTimeout = 500 * time.Millisecond
+const (
+	// env name to make firecracker request timeout configurable
+	firecrackerRequestTimeoutEnv = "FIRECRACKER_GO_SDK_REQUEST_TIMEOUT_MILLISECONDS"
+
+	defaultFirecrackerRequestTimeout = 500
+)
+
+var (
+	firecrackerRequestTimeout, firecrackerInitTimeout int
```

Comment:
I'd much prefer to not have global/package-level variables when we can avoid it.  Can you move these into the `Client` struct?